kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/KafkaServiceConfiguration.java

```diff
@@ -245,9 +245,9 @@
 
     @FieldContext(
             category = CATEGORY_KOP,
-            doc = "Maximum number of entries that are read from cursor once per time"
+            doc = "Maximum number of entries that are read from cursor once per time, default is 1"
     )
-    private int maxReadEntriesNum = 5;
+    private int maxReadEntriesNum = 1;
```

Comment:
Is the change here necessary?

Reply:
Yes, this fix the problem of CommitOffsetBacklogTest.testOffsetCommittedBacklogCleared